example/trivial/datadog/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:18.04 
```

Comment:
Could just use the golang containers ( https://hub.docker.com/_/golang/ ) with whichever base and Go version you want.